srvutil/metrics_test.go

```diff
@@ -71,9 +71,10 @@ func TestRequestMetricsMiddleware(t *testing.T) {
 	assert.Equal(t, []string{"route:/hello/@name", "route_name:world", "statusClass:2xx", "statusCode:200", "success:true"}, recordedTags)
 
 	output := strings.ToLower(logging.String())
-	assert.Contains(t, output, "foo: bar\\r\\n")
-	assert.Contains(t, output, "foo: baz\\r\\n")
-	assert.NotContains(t, output, "authorization: secret\\r\\n")
-	assert.NotContains(t, output, "cookie: secret\\r\\n")
-	assert.NotContains(t, output, "set-cookie: secret\\r\\n")
+	assert.Contains(t, output, "foo:bar")
```

Comment:
I don't understand why these have changed.

Reply:
Because previously we logged in the format of a request/response header (`\r\n`-delimited text):

```
Foo: bar\r\n
```

But now we log as a map.